provider/units/laptop.pxu

```diff
@@ -7,4 +7,6 @@ nested_part:
     hotkey
     manual
     automated
+    suspend
+    manual_after_suspend
```

Comment:
The hotkey tests should be run after suspend, in a seperate job such as hotkey_after_suspend